Documentation/blobIn.md

```diff
@@ -1,3 +1,12 @@
+#### Connecting to your Storage Account
+
+* Download Storage explorer from here: http://storageexplorer.com/ 
+* Connect using these credentials:
+* Account Name: `{{accountName}}`
+* Account Key: `{{accountKey}}`
```

Comment:
BTW I noticed we do display github secret in plain text
![image](https://cloud.githubusercontent.com/assets/2650941/18887856/d811200c-84aa-11e6-926a-fbcc18e83331.png) . Still dont have a good answer on hiding this with markdown. Working on it